TreasureHunt/Assets/2.Scripts/MakerSceneManager.cs

```diff
@@ -28,8 +28,13 @@ public void uploadToCloud()
 		camera.targetTexture = rt;
 		camera.Render();
 		RenderTexture.active = rt;
-		Rect temp = new Rect (0, (height - width)/2, width, width);
-		tex.ReadPixels(temp, 0, 0);
+
+		/* hyunSo
+		 * variable cropImageRect for cropping image boundary
+		 * for reducing image file size and decrease useless recognition features
+		 */
```

Comment:
In this case, we use camera image directly so I don' think we confront with extremely big image. But it seems to be great if we can know and restrict the image size... I'll add it as a TODO. Thank you.